pkg/assembler/backends/inmem/hasSBOM.go

```diff
@@ -153,6 +156,7 @@ func (c *demoClient) ingestHasSbom(ctx context.Context, subject model.PackageOrA
 		downloadLocation: input.DownloadLocation,
 		origin:           input.Origin,
 		collector:        input.Collector,
+		knownSince:       input.KnownSince,
```

Comment:
nit: should we store as UTC for all the timestamps to be consistent with scorecard and other verbs?

Reply:
I have changed all the time stamps to UTC.